modules/node/src/cfCore/cfCore.store.spec.ts

```diff
@@ -335,7 +335,7 @@ describe("CFCoreStore", () => {
   });
 
   describe("App Instance", () => {
-    it("should not create an app instance if there is no app proposal", async () => {
+    it.skip("should not create an app instance if there is no app proposal", async () => {
```

Comment:
If this was on purpose, let's add a comment explaining what needs to happen before we unskip this test.